Rakefile

```diff
@@ -4,3 +4,7 @@
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
+
+require 'rubocop/rake_task'
+
+RuboCop::RakeTask.new
```

Comment:
This is cool. But does it do anything that running 'rubocop' from the command line doesn't do?